### PR TITLE
Document catalog configuration added to storeConfig query

### PR DESCRIPTION
**Pull request by pmclain**
## Purpose of this pull request

This PR adds catalog configuration elements available in the `storeConfig` graphql query. These elements were added in magento/graphql-ce#650

Fixes magento/graphql-ce#768

## Links to Magento source code

https://github.com/magento/graphql-ce/blob/9f2f5b605564465876b391883be3f19035b5f80a/app/code/Magento/CatalogGraphQl/etc/graphql/di.xml#L83-L97

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/graphql/reference/store-config.md [hunk 1]
```diff
@@ -80,6 +80,22 @@ Attribute |  Data Type | Description
 `no_route` | String | Contains the URL of the default page that you want to appear when if a 404 “Page not Found” error occurs
 `show_cms_breadcrumbs` | Int | Determines if a breadcrumb trail appears on all CMS pages in the catalog. Options: `0` (No) or `1` (Yes)
 
+### Supported Catalog attributes
+
+Use the `catalog` attributes to retrieve information about the store's catalog. These attributes are defined in the `CatalogGraphQl` module.
+
```
Comment (keharper) [inline guides/v2.3/graphql/reference/store-config.md]: General comments about this table.

* Add a fourth column to the header and separator rows (lines 87-88). The fourth column should be labeled "Example".
* Alphabetize the attributes

**Comment by keharper** [comment]
running tests

**Comment by meker12** [comment]
@keharper Missing `whatsnew` required for major updates.

**Pull request merged**